src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpMetricExporterExtensions.cs

```diff
@@ -25,48 +25,100 @@ namespace OpenTelemetry.Metrics
         /// <returns>The instance of <see cref="MeterProviderBuilder"/> to chain the calls.</returns>
-        public static MeterProviderBuilder AddOtlpExporter(this MeterProviderBuilder builder, Action<OtlpExporterOptions> configure = null)
+        public static MeterProviderBuilder AddOtlpExporter(
+            this MeterProviderBuilder builder,
+            Action<OtlpExporterOptions, MetricReaderOptions> configureExporterAndMetricReader)
         {
             Guard.ThrowIfNull(builder, nameof(builder));
 
             if (builder is IDeferredMeterProviderBuilder deferredMeterProviderBuilder)
             {
                 return deferredMeterProviderBuilder.Configure((sp, builder) =>
                 {
-                    AddOtlpExporter(builder, sp.GetOptions<OtlpExporterOptions>(), configure, sp);
+                    AddOtlpExporter(builder, sp.GetOptions<OtlpExporterOptions>(), sp.GetOptions<MetricReaderOptions>(), null, configureExporterAndMetricReader, sp);
                 });
             }
 
-            return AddOtlpExporter(builder, new OtlpExporterOptions(), configure, serviceProvider: null);
+            return AddOtlpExporter(builder, new OtlpExporterOptions(), new MetricReaderOptions(), null, configureExporterAndMetricReader, serviceProvider: null);
         }
 
         private static MeterProviderBuilder AddOtlpExporter(
             MeterProviderBuilder builder,
-            OtlpExporterOptions options,
-            Action<OtlpExporterOptions> configure,
+            OtlpExporterOptions exporterOptions,
+            MetricReaderOptions metricReaderOptions,
+            Action<OtlpExporterOptions> configureExporter,
+            Action<OtlpExporterOptions, MetricReaderOptions> configureExporterAndMetricReader,
             IServiceProvider serviceProvider)
         {
-            var initialEndpoint = options.Endpoint;
+            var initialEndpoint = exporterOptions.Endpoint;
 
-            configure?.Invoke(options);
+            if (configureExporterAndMetricReader != null)
+            {
+                configureExporterAndMetricReader.Invoke(exporterOptions, metricReaderOptions);
+            }
+            else
+            {
+                configureExporter?.Invoke(exporterOptions);
+            }
+
+            exporterOptions.TryEnableIHttpClientFactoryIntegration(serviceProvider, "OtlpMetricExporter");
+
+            exporterOptions.AppendExportPath(initialEndpoint, OtlpExporterOptions.MetricsExportPath);
 
-            options.TryEnableIHttpClientFactoryIntegration(serviceProvider, "OtlpMetricExporter");
+            var metricExporter = new OtlpMetricExporter(exporterOptions);
 
-            options.AppendExportPath(initialEndpoint, OtlpExporterOptions.MetricsExportPath);
+            if (metricReaderOptions.Temporality == (AggregationTemporality)0)
+            {
+                metricReaderOptions.Temporality = AggregationTemporality.Cumulative;
+            }
```

Comment:
I'm not thrilled by this way of handling default values. Open to other options.

Reply:
I'm running under the assumption that for the OTLP exporter we're going to want to default to periodic. But for the console exporter we'll want to default to manual, so a uniform default will not work for all exporters. `MetricReaderType == -1` helps distinguish that the user did not explicitly configure this option.

Reply:
Oh, oops, you're commenting on Temporality, yea I guess you're right, Cumulative can be considered the default always right?